### ISSUE TITLE
[BUG] Playwright Inspector exits after 30 seconds if run with page.pause()

### ISSUE THREAD

**Issue by andrejs-ps** [microsoft/playwright#10353]
**Context:**
- Playwright Version: 1.16
- Operating System: Windows
- Node.js version: 14.18.1
- Browser: All

**Overview**
There are multiple ways to turn on the Inspector, as explained in https://playwright.dev/docs/inspector
As per docs, when we set "PWDEBUG", it also disables the timeout, by setting it to 0. This is a good idea, as it allows the user to click around the inspector without a time limit.

However, this is not the case with `page.pause()`. This option also opens the inspector, but the timeout is not set to 0, thus the inspector will force exit after 30s with e.g. 

` Slow test:  tests\my_test.test.js (30s)`

The same behavior is mentioned in a Feature request here: https://github.com/microsoft/playwright/issues/10132
And I believe it should be fixed to match the default behaviour of other options, at least for consistency reasons. For example, in Java binding, `page.pause()` does not require timeOut to be explicitly disabled.

**Code Snippet**

```javascript
test.only('Inspector Demo', async ({ page }) => {
  test.setTimeout(0); // this helps, but I believe it shouldn't be needed
  await page.goto('https://playwright.dev');

  await page.pause();
});
```

**Comment by mariaceo** [comment]
Is this back now?
The inspector is automatically closing for me after 60s.
I tried:
-  PWDEBUG=1
- test.setTimeout(0)
- --timeout=0
- removed all timeouts from the config file
The inspector still closes automatically after 1 minute. Am I missing something or theres a problem with the timeout when debugging?

**Comment by dgozman** [comment]
@mariaceo This is most likely #12974, which was fixed in v1.20.1. If you upgrade, things should work.

**Comment by mariaceo** [comment]
> @mariaceo This is most likely #12974, which was fixed in v1.20.1. If you upgrade, things should work.

True that 👌  I bumped the version and it works as expected now, many thanks 🎉
